README.md

```diff
@@ -1,7 +1,6 @@
 # packtivity
 
 [![DOI](https://zenodo.org/badge/53696818.svg)](https://zenodo.org/badge/latestdoi/53696818)
-[![Build Status](https://travis-ci.org/yadage/packtivity.svg?branch=master)](https://travis-ci.org/diana-hep/packtivity)
 [![Coverage Status](https://coveralls.io/repos/github/diana-hep/packtivity/badge.svg)](https://coveralls.io/github/diana-hep/packtivity)
```

Comment:
The coverage URL will need to get updated too (currently still showing DIANA/HEP)